rover_antenna/src/arbitration.cpp

```diff
@@ -0,0 +1,104 @@
+#include "rclcpp/rclcpp.hpp"
+#include "rover_msgs/msg/antenna_cmd.hpp"
+#include "rover_msgs/srv/antenna_arbitration.hpp"
+
+using namespace std::chrono_literals;
+
+class Arbitration : public rclcpp::Node
+{
+public:
+    Arbitration();
+    ~Arbitration() {}
+
+private:
+    void timer_callback()
+    {
+        RCLCPP_INFO(rclcpp::get_logger("timer_cb"), "timer_callback");
```

Comment:
this->get_logger(), live tu renomme le nom du noeud. Le print sert pas à grand chose nn plus

Reply:
fait